index.html

```diff
@@ -4,360 +4,402 @@
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
               </p>
             </div>
-            <div class="project-tag-container">
-              <ul class="tag-item">
-                <li class="tags"><a class="tag-btn" href="#">html</a></li>
-                <li class="tags"><a class="tag-btn" href="#">css</a></li>
-                <li class="tags"><a class="tag-btn" href="#">javaScript</a></li>
+            <section class="connect-section">
+              <h3 class="link-item">
+                <a href="#" class="connect">LET'S CONNECT</a>
+              </h3>
+            </section>
+            <div class="social">
+              <ul class="icon-svgs">
+                <li class="icon"><a class="svg twitter" href="#"></a></li>
+                <li class="icon"><a class="svg linkedlin" href="#"></a></li>
+                <li class="icon"><a class="svg medium" href="#"></a></li>
+                <li class="icon"><a class="svg git" href="#"></a></li>
+                <li class="icon"><a class="svg hand" href="#"></a></li>
               </ul>
             </div>
-            <div class="projects-btn">
-              <input type="button" value="See Project" class="project-button" />
-            </div>
-          </div>
-         </div>
-        </div>
-        <div class="content work-projects work-two">
-          <div class="item project-header">
-            <div class="projectsimage">
-              <img
-                src="assets/images/portfolio-image3.PNG"
-                alt="project image title"
-                class="headimage"
-              />
-            </div>
           </div>
-          <div class="cards-description">
-          <div class="item project-description">
-            <div class="project-title-container">
-              <h4 class="description-title">Tonic</h4>
-            </div>
-            <div class="project-subtitle-container">
-              <ul class="subtitle-items">
-                <li class="list-less"><p class="listinfo">CANOPY</p></li>
-                <li class="list"><p>Back End Dev</p></li>
-                <li class="list"><p>2015</p></li>
-              </ul>
-            </div>
-            <div class="project-article-container">
-              <p class="description-item description-item-b">
-                A daily selection of privately perosonalized reads; no accounts
-                or sign-ups required.
-              </p>
-            </div>
-            <div class="project-tag-container">
-              <ul class="tag-item">
-                <li class="tags"><a class="tag-btn" href="#">html</a></li>
-                <li class="tags"><a class="tag-btn" href="#">css</a></li>
-                <li class="tags"><a class="tag-btn" href="#">javaScript</a></li>
-              </ul>
+        </section>
+        <section id="myworks">
+          <div class="content work-projects work-one">
```

Comment:
- [ ] The first project is a lot bigger than the others.  Kindly correct this so it is consistent with all the others, like in Figma.

## Explanation and Hints

Just make this project have the same height as all the others.